app/models/travel_advice.rb

```diff
@@ -0,0 +1,35 @@
+class TravelAdvice < MultipageContent
+
+  DETAILS_ATTRIBUTES = %i(
+    summary change_description alert_status email_signup_link
+  )
```

Comment:
Yes, I agree, it's an uncomfortable pattern for me too, was trying to escape `OpenStruct` but this ended up being quite ugly in itself. I wonder how much of this could be worked into presenters...